src/client/app/containers/CompareChartContainer.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import _ from 'lodash';
+import { Bar } from 'react-chartjs-2';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import datalabels from 'chartjs-plugin-datalabels';
+import GraphColors from '../utils/GraphColors';
+
+
+/**
+ * @param {State} state
+ */
+function mapStateToProps(state) {
+	const timeInterval = state.graph.compareTimeInterval;
+	const barDuration = state.graph.compareDuration;
+	const data = { datasets: [] };
+	const labelsSet = new Set();
+	let cw = 0;
+	let lw = 0;
+	let clw = 0;
+	const soFar = moment().diff(moment().startOf('week'), 'days');
+	for (const meterID of state.graph.selectedMeters) {
+		const readingsData = state.readings.bar.byMeterID[meterID][timeInterval][barDuration];
+		if (readingsData !== undefined && !readingsData.isFetching) {
+			// Calculate cw
+			for (let i = 1; i <= soFar; i++) {
+				cw += readingsData.readings[readingsData.readings.length - i][1];
+			}
+			// Calculate lw
+			for (let i = 0; i < 7; i++) {
```

Comment:
Could you add more comments to how you calculate cw, clw, and lw? Also maybe explain these abbreviations?